pallets/liquidity-pools/src/message.rs

```diff
@@ -199,6 +200,7 @@ where
 		pool_id: PoolId,
 		tranche_id: TrancheId,
 		investor: Address,
+		currency: u128,
```

Comment:
NOTE: This additions was discussed. It is required as long as we do not store the `return_currency` when the user increases their redemption.